src/applications/coronavirus-vaccination/config/uiSchema.js

```diff
@@ -23,11 +26,25 @@ export default {
   },
   birthDate: {
     'ui:title': 'Date of birth',
+    'ui:description': () => (
+      <span>
+        <b>Note: </b>
+        Your date of birth helps us match your information to your Veteran
+        records so we can better understand your needs.
+      </span>
+    ),
     'ui:widget': 'date',
   },
   ssn: {
     ...ssnUiSchema,
     'ui:title': 'Social Security number (SSN)',
+    'ui:description': () => (
+      <span>
+        <b>Note: </b>
+        Your SSN helps us match your information to your Veteran records so we
+        can better understand your needs.
+      </span>
+    ),
```

Comment:
In the `'ui:description'` fields,  a `<p>` might give us a little bit of top/bottom margin (I can't tell if it needs that or not though so your call)

Reply:
@DanielleThierryUSDSVA thoughts on spacing in the Note: info for SSN and DOB?